src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2629,6 +2632,12 @@ export class StudyViewPageStore {
         this._filterMutatedGenesTableByCancerGenes = true;
         this._filterFusionGenesTableByCancerGenes = true;
         this._filterCNAGenesTableByCancerGenes = true;
+
+        this._clinicalDataBinFilterSet.keys().forEach(chartUniqueKey => {
```

Comment:
i'm just curious why we wouldn't just clear the whole filter set here?

Reply:
this is initialized when clinical attributes are fetched and is required to fetch clinical data. Other option is to reset it to default(_defualtClinicalDataBinFilterSet)